napari/_app_model/actions/_layer_actions.py

```diff
@@ -50,27 +50,27 @@
         id=CommandId.LAYER_SPLIT_STACK,
         title=CommandId.LAYER_SPLIT_STACK.title,
         callback=_layer_actions._split_stack,
-        menus=[{**LAYERCTX_SPLITMERGE, 'when': ~LLCK.active_layer_is_rgb}],
-        enablement=LLCK.active_layer_is_image_3d,
+        menus=[{**LAYERCTX_SPLITMERGE, 'when': ~LLSCK.active_layer_is_rgb}],
+        enablement=LLSCK.active_layer_is_image_3d,
```

Comment:
Detailed in first bullet here: https://github.com/napari/napari/pull/4865#issuecomment-1635711551

Yes we could remove from this PR. I would leave the `DLG_SAVE_LAYERS` action without an enablement (i.e you would always be able to click on it and it would just return an error if no layers), then in another PR I could add the add the Layer context and add the enablement. It was added in this PR in order to make the `DLG_SAVE_LAYERS` work.

Reply:
Done in #6165